composer.json

```diff
@@ -19,13 +19,13 @@
         "doctrine/orm": "^2.5.3",
         "doctrine/annotations": "^1.2",
         "doctrine/phpcr-bundle": "^1.3.5",
-        "doctrine/doctrine-bundle": "~1.0",
+        "doctrine/doctrine-bundle": "~1.4",
```

Comment:
else content bundle some test fails between (63 - 123) (mostly a Too many connections error). Here I'm not sure if this is really needed to run sulu as the tests work when it is run as single test and not the whole bundle. without this change running the whole content bundle and running tests on travis will not work. /cc @danrot